app/main.py

```diff
@@ -1,2 +1,87 @@
+from dataclasses import dataclass
+from typing import Hashable, Iterator
+
+
+@dataclass
+class Node:
+    key: Hashable
+    value: any
+    hash_value: int
+
+
 class Dictionary:
-    pass
+    INITIAL_CAPACITY = 8
+    THRESHOLD = 2 / 3.0
```

Comment:
The `THRESHOLD` is set to `2 / 3.0`, which is a float division. However, it should be noted that the threshold should be a fraction (e.g., `0.75`) to represent the load factor limit. Ensure that this value is intended as a fraction and not an integer division.